### PR TITLE
parsed memory info w/ example, name change

### PR DESCRIPTION
**Pull request by lydiachoi**

### CODE REVIEW

File: parsers/memory.awk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+BEGIN{
+  ORS=",";
+  print ",Commit, (Pass/fail, Test Class, Heap Usage in MB)* \n";
```
Comment (jregchiu) [inline parsers/memory.awk]: Not sure what we need in terms of info for the visualization (@xavierdelacruz or @shannonjanehogan can comment on that) but I think the header should look like `Commit, Pass/fail, Test Class, Heap Usage in MB` without the leading comma or parentheses

**Review by jregchiu — CHANGES_REQUESTED** [review]

File: parsers/memory.awk (hunk 1 above)
Comment (xavierdelacruz) [reply parsers/memory.awk]: @lydiachoi, for headers, we don’t want to have spaces between commas. For example, the header would be Commit,Pass/Fail,Test Class,Heap Usage in MB

Comment (lydiachoi) [reply parsers/memory.awk]: will change the spacing, but the leading comma gets edited out in the final csv!

**Review by xavierdelacruz — APPROVED** [review]
Looks good, let’s see what Shannon comes up with the dynamic one

File: compute-testing-info [hunk 2]
```diff
@@ -66,7 +66,7 @@ do
 
         # output test coverage and memory usage information of commit to file
         bash ./get-coverage-and-memory $TEMP_DIR 2>&1 | tee -a $RAW_OUTPUT
-        bash ./get-changed-test-files $TEMP_DIR 2>&1 | tee -a changedTestFiles.txt
+        # bash ./get-changed-test-files $TEMP_DIR 2>&1 | tee -a changedTestFiles.txt
```
Comment (lydiachoi) [inline compute-testing-info:69]: taking this out we might not have time to get to it so let's park it so it doesn't waste that much time

**Review by jregchiu — APPROVED** [review]

**Pull request merged**